x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/chat/support-template.json

```diff
@@ -0,0 +1,67 @@
+{
+  "index_patterns": ["content-chat-support-v1-*"],
+  "template": {
+    "mappings": {
+      "properties": {
+        "metadata": {
+          "properties": {
+            "support": {
+              "properties": {
+                "status": {
+                  "type": "keyword"
+                },
+                "assignee": {
+                  "properties": {
+                    "name": {
+                      "type": "text",
+                      "fields": {
+                        "keyword": {
+                          "type": "keyword"
+                        }
+                      }
+                    },
+                    "email_address": {
+                      "type": "keyword"
+                    },
+                    "id": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "category": {
+                  "type": "keyword"
+                },
+                "customer": {
+                  "properties": {
+                    "name": {
+                      "type": "text",
+                      "fields": {
+                        "keyword": {
+                          "type": "keyword"
+                        }
+                      }
+                    },
+                    "email_address": {
+                      "type": "keyword"
+                    },
+                    "id": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "resolved_at": {
+                  "type": "date"
+                },
+                "urgency": {
+                  "type": "keyword"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "composed_of": ["base-template", "comments-template"],
```

Comment:
Do you expect base-template to expand and cover data types such as customer data, docs/knowledge articles? Or will we add new templates for each of the relevant data types e.g. https://docs.google.com/document/d/1_4wVdQnY_O-Mcw10UzIcOHvYRdjZmy57xFqN34-bBmA/edit?tab=t.fcn3f8i8o2u3#heading=h.ryizjiyrh303

Reply:
Both. Hopefully the base template changes less, but I think we should expect that it'll grow.